java/arcs/android/storage/service/StorageServiceManagerBindingContext.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.android.storage.service
+
+import kotlin.coroutines.CoroutineContext
+
+/**
+ * A [StorageServiceManagerBindingContext] is used by a client of the [StorageService] to manage
+ * data stored within the [StorageService].
+ */
+class StorageServiceManagerBindingContext(
```

Comment:
The term `BindingContext` is really only used for the store-functionality of the storage service. It'd be fine to call this `StorageServiceManager`